src/components/layout/tables/AntTable/styles.css

```diff
@@ -0,0 +1,40 @@
+.assets-table {
+  display: flex;
+  flex-direction: column;
+  row-gap: 1em;
+  width: 100%;
+}
+
+.assets-table-header {
+  display: flex;
+  position: relative;
+  flex-direction: row;
+  height: 50px;
+  width: 100%;
+  border-radius: 6px;
+  background-color: #323232;
+  align-items: center;
+}
+.assets-table-item {
+  display: flex;
+  flex-direction: row;
+  position: absolute;
+  justify-content: center;
+  align-items: center;
+  color: var(--text-white);
+}
+.assets-table-item:nth-child(1) {
+  left: 50px;
+}
+.assets-table-item:nth-child(2) {
+  left: 25%;
+}
+.assets-table-item:nth-child(3) {
+  left: 50%;
+}
+.assets-table-item:nth-child(4) {
+  left: 75%;
+}
+.assets-table-item:nth-child(5) {
+  right: 10px;
+}
```

Comment:
can we use `width:auto` for the `<td>` and set max-widths instead of hard coding these?

Reply:
hm. Maybe im not doing this right, i wound up with this 
![image](https://user-images.githubusercontent.com/85306700/213009924-2a84ec30-f392-4c83-9664-ad3a9df3a959.png)

Reply:
the reason i hard coded it like that is to make all the column data items line up with the column headers consistently. If the length of a table item changes it becomes out of line with the others otherwise. There might be a different solution, but I think it would involve potentially more hard coding. Or inline css w/ jsx maybe